packages/vite/src/node/utils.ts

```diff
@@ -598,6 +598,10 @@ export function copyDir(srcDir: string, destDir: string): void {
       continue
     }
     const destFile = path.resolve(destDir, file)
+    // This may occur if the source directory is a symlink that points to a non-existent directory
+    if (!fs.existsSync(destFile)) {
```

Comment:
I think this should be
```suggestion
    if (!fs.existsSync(srcFile)) {
```
Also I think it'd nice to avoid this `existsSync` call and add an `try-catch` to `fs.statSync` instead. `fs.statSync` throwed `ENOENT` on my machine (both Windows and WSL), did `fs.statSync` return a result on your machine?